csaf_2.0/json_schema/csaf_json_schema.json

```diff
@@ -418,6 +420,7 @@
             },
             "generator": {
               "type": "object",
+              "minProperties": 1,
```

Comment:
The generator might be private to the producer. Not sure we should enforce in the public API - this may lead to disguised or faked or ignored fields in practice. But I have no hard feelings about this.

Reply:
> 
> 
> The generator might be private to the producer. Not sure we should enforce in the public API - this may lead to disguised or faked or ignored fields in practice. But I have no hard feelings about this.

In this case the `generator` property can be completely removed from the document. I prefer to remove this redundancy, so I would approve this change.

Reply:
Well if some organization wants to publish the generator, I guess they should be able. Others can simply not provide the member and value.